# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxHarfBuzzShaper.h
# Commit: 7bacec3b6ce1
# Full Hash: 7bacec3b6ce17faf630b808ffbac94ce6f125c78
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 5 - Implement support for COLRv1 glyphs represented as acyclic graphs of paint records. r=gfx-reviewers,lsalzman
#   
#   Depends on D153870
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152041
# ==============================================================================

diff -r 4354a0d7248f -r 7bacec3b6ce1 gfx/thebes/gfxHarfBuzzShaper.h
--- a/gfx/thebes/gfxHarfBuzzShaper.h	Wed Aug 17 10:59:34 2022 +0000
+++ b/gfx/thebes/gfxHarfBuzzShaper.h	Wed Aug 17 10:59:35 2022 +0000
@@ -100,6 +100,9 @@
                                  hb_font_funcs_t* aFontFuncs = nullptr,
                                  FontCallbackData* aCallbackData = nullptr);
 
+  hb_font_t* GetHBFont() const { return mHBFont; }
+  hb_face_t* GetHBFace() const { return hb_font_get_face(mHBFont); }
+
  protected:
   nsresult SetGlyphsFromRun(gfxShapedText* aShapedText, uint32_t aOffset,
                             uint32_t aLength, const char16_t* aText,